.gitignore

```diff
@@ -25,4 +25,15 @@ test/version_tmp
 tmp
 vendor/
 /easycop.yml
-/.rubocop.yml
\ No newline at end of file
+/.rubocop.yml
+[._]*.s[a-v][a-z]
+[._]*.sw[a-p]
+[._]s[a-rt-v][a-z]
+[._]ss[a-gi-z]
+[._]sw[a-p]
+Session.vim
+Sessionx.vim
+.netrwhist
+*~
+tags
+[._]*.un~
```

Comment:
what in the what is all this? 😂

Reply:
Ignore rules for various `vim` stuff, mainly swap and session files.

Reply:
I would consider setting up something like this in your vim config (what I did for emacs): https://github.com/Justintime50/dotfiles/blob/main/src/emacs/.emacs#L96-L102

This way, you won't have these spread out all over your file system. Pushing all those tmp files to a tmp dir for vim really cleaned that up for me.

Reply:
`vim` deletes them after you close the file you're editing. Having these rules benefit those who don't have configured their `vim`/`emacs` the way you have and don't have to worry about checking in unwanted files by default :D